plugins/user.py

```diff
@@ -84,6 +84,23 @@ class UserAddExtraFields(UserAdd):
     manual = BooleanField(label="Usuario Manual", validators=[DataRequired()], render_kw={'checked': True})
     person_type = SelectField(label="Tipo de Persona", choices=[('Worker', "Trabajador"), ('Student', "Estudiante")])
     dni = StringField(label='Carné Identidad', validators=[DataRequired(), Length(min=11,max=11)])
+    i=0
+    for extra in Settings.extra_fields:
+        if extra[2] is 'boolean':  #extra[0]:label, extra[1]:ldap field, extra[2]: data type, extra[3]:choices
```

Comment:
We need the explanatory comment anyway since we need to use the diferent items in extra for specific purposes below, and I was preventing that 'boolean' for example was specifically entered as a data type instead of a label or ldap field (which maybe could happened).

Reply:
Yeah, but still, I think code will be far more clear with the change I proposed, its seems more intuitive about what's going on, and it allows you to move things around if you need to.